javascript/modules/client/src/internal/interfaces/dao.ts

```diff
@@ -19,9 +14,10 @@ export interface IClientDaoBase extends IClientCore {
   };
 }
 
-export interface IClientDaoWhitelist extends IClientCore {
+export interface IClientDaoERC20Voting extends IClientCore {
   dao: {
-    whitelist: {
+    create: (params: ICreateDaoERC20Voting) => Promise<string>;
```

Comment:
I see that we are dropping the sub-namespaces. Then, is the proposal to do `client.dao.myRelevantCustomOperation()` instead?